test/helpers/holiday_helpers.rb

```diff
@@ -11,6 +11,14 @@ class MissingConfigurationError < RuntimeError; end
         { month: 11, day: 24 },
         { month: 12, day: 26 },
       ],
+      "2017" => [
+        { month: 1,  day: 3  },
```

Comment:
Although the 2017 Holiday Schedule hasn't been released yet, this says January 2nd is a Holiday in 2017: https://compass.ups.com/ups-holiday-calendar/